lib/DDG/Spice/SentimentAnalysis/Entities.pm

```diff
@@ -0,0 +1,25 @@
+package DDG::Spice::SentimentAnalysis::Entities;
+# ABSTRACT: Returns result of sentiment analysis over a text or online content
+
+use DDG::Spice;
+
+attribution github  => ['https://github.com/gpestana', 'gpestana'],
+            twitter => ['https://twitter.com/gpestana', 'gpestana'];
+
+spice is_cached => 1;
+spice proxy_cache_valid => "200 1d";
+spice wrap_jsonp_callback => 1;
+
+spice to => 'http://gateway-a.watsonplatform.net/calls/url/URLGetRankedNamedEntities?apikey={{ENV{DDG_SPICE_SENTIMENTANALYSIS_APIKEY}}}&url=$1&outputMode=json';
+
+triggers start => '///***never_triggers***///';
```

Comment:
@gpestana I have good news -- we *just* merged some changes today. You now longer need a separate package for a second API endpoint. There is a new `spice alt_to =>` hash which can be used from a single package. This should simplify your code :)

Checkout this PR for examples: https://github.com/duckduckgo/zeroclickinfo-spice/pull/2308

The other IA's in the repo have now been updated